docs/root/configuration/http/http_filters/decompressor_filter.rst

```diff
@@ -53,6 +53,9 @@ When decompression is *applied*:
 
 - The *content-encoding* header is modified to remove the decompression that was applied.
 
+- *x-envoy-decompressor-<decompressor_name>-<compressed/uncompressed>-bytes* trailers are added to
+  the request/response to relay information about decompression.
```

Comment:
Yeah, this can be added to the compressor too, will do in a subsequent PR, and only when we (lyft) wants to start compressing requests using envoy mobile.